### DISCUSSION TITLE
Why can't I get Date Note to work?

### DISCUSSION THREAD

**Discussion by ghost** [SkepticMystic/breadcrumbs#374] [Q&A]
Hi, I've been loving the plugin since I discovered it (yesterday!) but I have trouble understanding how to use the Date Notes feature.
Simple scenario:
- Daily Notes all in one giant folder, title for the notes is YYYY-MM-DD
- I set Date Notes to the required format, which *I think* should be: YYYY-MM-dd
- I don't get any result out of this. I tried some other formats to no avail.
Note: My real-usage scenario would be more complicated with year and month subfolders, like so: Journals/YYYY/MM/DD MMM YY and the "Override locale" Calendar option should be set to my locale (specifically, "it"), returning e.g. Journals/2022/01/01 Gen 22
Right now I don't understand if it's just me not understanding what format to use or there's something more to it. (I don't know anything about programming and I couldn't really follow your info on debugging, sorry). Thanks for any help!

**Comment by michaelpporter** [comment]
This was reported against the v3 Date Notes feature back in 2022. Date Notes was rebuilt in v4 and works well now — the most common cause of "no result" is the format string.

v4 uses **Luxon** format tokens, not Moment.js. So a `YYYY-MM-DD` filename is matched with lowercase tokens:

- Date format: `yyyy-MM-dd` (not `YYYY-MM-DD`)

Enable it under **Settings → Date Notes**, set the format to match your filenames, and sequential daily notes are linked automatically via `next`/`prev`. Notes in subfolders are fine — matching is on the basename, so one big folder or `Journals/YYYY/MM/…` both work. For the enclosing year/month buckets and localized month names, use the per-period (week/month/quarter/year) configs.

Closing as outdated (v3-era). If v4 Date Notes still doesn't pick up your format, please open a fresh issue with your exact filename and format string. 🍞
